app/App/HomeController.php

```diff
@@ -18,49 +18,49 @@
 class HomeController extends Controller
 {
     /**
-     * Display the homepage.
-     */
+    * Display the homepage.
+    */
     public function index(Request $request, ActivityQueries $activities)
     {
         $activity = $activities->latest(10);
         $draftPages = [];
-
+        
         if ($this->isSignedIn()) {
             $draftPages = Page::visible()
-                ->where('draft', '=', true)
-                ->where('created_by', '=', user()->id)
-                ->orderBy('updated_at', 'desc')
-                ->with('book')
-                ->take(6)
-                ->get();
+            ->where('draft', '=', true)
+            ->where('created_by', '=', user()->id)
+            ->orderBy('updated_at', 'desc')
+            ->with('book')
+            ->take(6)
+            ->get();
         }
-
+        
         $recentFactor = count($draftPages) > 0 ? 0.5 : 1;
         $recents = $this->isSignedIn() ?
-            (new RecentlyViewed())->run(12 * $recentFactor, 1)
-            : Book::visible()->orderBy('created_at', 'desc')->take(12 * $recentFactor)->get();
+        (new RecentlyViewed())->run(12 * $recentFactor, 1)
+        : Book::visible()->orderBy('created_at', 'desc')->take(12 * $recentFactor)->get();
         $favourites = (new TopFavourites())->run(6);
         $recentlyUpdatedPages = Page::visible()->with('book')
-            ->where('draft', false)
-            ->orderBy('updated_at', 'desc')
-            ->take($favourites->count() > 0 ? 5 : 10)
-            ->select(Page::$listAttributes)
-            ->get();
-
+        ->where('draft', false)
+        ->orderBy('updated_at', 'desc')
+        ->take($favourites->count() > 0 ? 5 : 10)
+        ->select(Page::$listAttributes)
+        ->get();
+        
```

Comment:
I think it'd be better to remove these spaces, as these can potentially break stuff in the future